src/main/java/io/dropwizard/jersey/protobuf/ProtocolBufferMessageBodyProvider.java

```diff
@@ -42,7 +47,10 @@ public Message readFrom(final Class<Message> type, final Type genericType,
             final InputStream entityStream) throws IOException {
 
         try {
-            final Method newBuilder = type.getMethod("newBuilder");
+            Method newBuilder = newBuilderMethodCache.get(type);
```

Comment:
what is adding items into the cache?

Reply:
Damn! forgot that line. Just pushed it now.

Reply:
is it possible to use `.computeIfAbsent()` here?